tests/e2e/test_methods/vanilla.py

```diff
@@ -0,0 +1,264 @@
+
+    resp = get_katib_experiment(cluster, region, namespace, name)
+
+    assert resp["kind"] == "Experiment"
+    assert resp["metadata"]["name"] == name
+    assert resp["metadata"]["namespace"] == namespace
+
+    resp = delete_katib_experiment(cluster, region, namespace, name)
+
+    assert resp["kind"] == "Experiment"
+    assert resp["metadata"]["name"] == name
+    assert resp["metadata"]["namespace"] == namespace
+
+    try:
+        get_katib_experiment(cluster, region, namespace, name)
+        raise AssertionError("Expected K8sApiException Not Found")
+    except K8sApiException as e:
+        assert "Not Found" == e.reason
+
+
+def test_ack_crds(
+    notebook_server,
+    framework_name,
+    ipynb_notebook_file,
+    expected_output,
+    user_namespace=DEFAULT_USER_NAMESPACE,
+):
+    """
+    Spins up a DLC Notebook and checks that the basic ACK CRD is installed.
+    """
+    nb_list = subprocess.check_output(
+        f"kubectl get notebooks -n {user_namespace}".split()
+    ).decode()
+
+    metadata_key = f"{framework_name}-notebook_server"
+    notebook_name = notebook_server["NOTEBOOK_NAME"]
+    assert notebook_name is not None
+    assert notebook_name in nb_list
+    print(notebook_name)
+
+    sub_cmd = f"jupyter nbconvert --to notebook --execute ../uploaded/{ipynb_notebook_file} --stdout"
+    cmd = f"kubectl -n kubeflow-user-example-com exec -it {notebook_name}-0 -- /bin/bash -c".split()
+    cmd.append(sub_cmd)
+
+    output = subprocess.check_output(cmd, stderr=subprocess.STDOUT).decode()
+    print(output)
+    # The second condition is now required in case the kfp test runs before this one.
+    assert expected_output in output or "training-job-" in output
+
+
+def test_run_kfp_sagemaker_pipeline(
+    kfp_client,
+    s3_bucket_with_data,
+    sagemaker_execution_role_arn,
+    user_namespace=DEFAULT_USER_NAMESPACE,
+):
+    random_prefix = rand_name("kfp-")
+
+    experiment_name = "experiment-" + random_prefix
+    experiment_description = "description-" + random_prefix
+    bucket_name = s3_bucket_with_data
+
+    job_name = "kfp-run-" + random_prefix
+
+    experiment = kfp_client.create_experiment(
+        experiment_name,
+        description=experiment_description,
+        namespace=user_namespace,
+    )
+
+    pipeline_id = kfp_client.get_pipeline_id(PIPELINE_SAGEMAKER_TRAINING)
+
+    params = {
+        "sagemaker_role_arn": sagemaker_execution_role_arn,
+        "s3_bucket_name": bucket_name,
+    }
+
+    run = kfp_client.run_pipeline(
+        experiment.id, job_name=job_name, pipeline_id=pipeline_id, params=params
+    )
+
+    assert run.name == job_name
+    assert run.pipeline_spec.pipeline_id == pipeline_id
+    assert run.status == None
+
+    wait_for_run_succeeded(kfp_client, run, job_name, pipeline_id)
+
+    kfp_client.delete_experiment(experiment.id)
+
+    cmd = "kubectl delete trainingjobs --all -n kubeflow-user-example-com".split()
```

Comment:
this is a problem in existing tests as well, this should be in a finally block so it runs irrespective of test pass/fail

Can we fix this now?

Reply:
done